lib/pbench/test/unit/agent/task/test_results_push.py

```diff
@@ -58,7 +58,6 @@ def test_missing_arg():
             args=[
                 TestResultsPush.TOKN_SWITCH,
                 TestResultsPush.TOKN_TEXT,
-                TestResultsPush.CTRL_TEXT,
```

Comment:
You should probably remove the definition of `CTRL_TEXT` at the top to be sure nobody uses it!